### PR TITLE
added support for auto-merge in nailgun repo

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
Added support for automerge in the Nailgun repo

### CODE REVIEW

File: .github/workflows/automerge.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  check_suite:
+    types:
+      - completed
+    branches-ignore:
+      - master
+
+
+jobs:
+  automerge:
+    name: Automerge auto-cherry-picked pr
```
Comment (Gauravtalreja1) [inline .github/workflows/automerge.yaml:25]: In Nailgun `Merging can be performed automatically with 2 approving reviews.`, are we planning to disable this first for auto-merge?

Comment (omkarkhatavkar) [reply .github/workflows/automerge.yaml:25]: @jyejare we can remove mentioned checks from nailgun for z-stream cherry picks.

Comment (jyejare) [reply .github/workflows/automerge.yaml:25]: @omkarkhatavkar @Gauravtalreja1 2 Approvals rule is not applicable for zStream branches. Its only for master.

**Comment by omkarkhatavkar** [comment]
@jyejare @Gauravtalreja1 can we review this as the it is already merged in the master for robottelo

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo

**Review by jyejare — APPROVED** [review]

**Pull request merged**